meta-balena-common/recipes-core/initrdscripts/files/abroot

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+abroot_enabled() {
+    if [ "$bootparam_balena_stage2" != "true" ]
+    then
+        return 1
+    fi
+
+    if [ "$bootparam_flasher" = "true" ]
+    then
+        return 1
+    fi
+
+    return 0
+}
+
+abroot_run() {
+    # Switch between rootA and rootB
+    export bootparam_root=LABEL=resin-rootA
```

Comment:
The switch is actually implemented later on - squashing up the latter commit here would make it easier on revierwers.